pkg/server/validation.go

```diff
@@ -31,7 +31,7 @@ func Validation(clients *clients.Clients) (http.Handler, error) {
 		prtbs := projectroletemplatebinding.NewValidator(clients.Management.RoleTemplate().Cache(), clients.EscalationChecker)
 		crtbs := clusterroletemplatebinding.NewValidator(clients.Management.RoleTemplate().Cache(), clients.EscalationChecker)
 		roleTemplates := roletemplate.NewValidator(clients.EscalationChecker)
-		provisioningCluster := cluster.NewProvisioningClusterValidator(clients.K8s.AuthorizationV1().SubjectAccessReviews())
+		provisioningCluster := cluster.NewProvisioningClusterValidator(clients)
```

Comment:
Do we only create the provisioning cluster if MCM is enabled?

Reply:
According to what I am seeing in the rancher/rancher code: no, the provisioning clusters are created for v3.Clusters regardless of whether MCM is enabled.

Reply:
So this would not work if MCM is disabled

Reply:
Could you explain why so that I might find another solution?

Reply:
Yeah, this is inside a `if clients.MultiClusterManagement {` check, so if mcm is disabled we don't register these handlers.

Reply:
Is the solution as simple as moving the relevant code outside the if-block?

Reply:
Yes.

Reply:
I tested the webhook with moving this outside of the if-block and it works as expected when MCM is disabled.